README.md

```diff
@@ -140,49 +140,41 @@ Start the server:
 
 `cargo r --bin iggy-server`
 
-Start the CLI (transports: `quic`, `tcp`, `http`):
+Create a stream with ID 1 named `dev` using default credentials and `tcp` transport (available transports: `quic`, `tcp`, `http`, default `tcp`):
```

Comment:
Please mention that all below commands are available via `iggy` binary, which is part of release (`iggy-cmd`).